tsl/src/nodes/decompress_chunk/exec.c

```diff
@@ -38,7 +38,8 @@ typedef struct DecompressChunkColumnState
 {
 	DecompressChunkColumnType type;
 	Oid typid;
-	AttrNumber attno;
+	AttrNumber output_attno;
+	AttrNumber compressed_scan_attno;
```

Comment:
Could you add comments on what these fields represent and are used for? It will make the job easier for the next person.

Reply:
Done